server/httpRequest.h

```diff
@@ -0,0 +1,32 @@
+#ifndef HTTPREQUEST_H
+#define HTTPREQUEST_H
+
+#include <string>
+#include <map>
+class HttpRequest{
+public:
+	bool Parse(const std::string request);
+	std::string getMessageBody();
+	std::string getMethod();
+	std::string getResourcePath();
+	std::string getVersion();
+	std::string getField(std::string header);
+
+	static const std::string GET;
+	std::map<std::string,std::string> header_fields;
+
+private:
+	bool processRequestLine(const std::string request);
```

Comment:
Are you sure you want these to be private? This makes unit testing difficult on these components.